forge/licensing/license-generator.js

```diff
@@ -61,6 +61,7 @@ const { v4: uuidv4 } = require('uuid')
         const maxUsers = parseInt(await promptly.prompt('Max allowed users: ', { default: '5' }))
         const maxTeams = parseInt(await promptly.prompt('Max allowed teams: ', { default: '5' }))
         const maxInstances = parseInt(await promptly.prompt('Max allowed instances (hosted + devices): ', { default: '5' }))
+        const maxMQTTClients = parseInt(await promptly.prompt('Max allowed MQTT Clients: ', { default: '20' }))
```

Comment:
That's 20 total for the platform, not just a give team, just making sure this is what we mean (It's probably right, just double checking)

Reply:
Yes - default enterprise license includes 20 clients; the license terms match the Enterprise team limits.